govcd/api_vcd_test.go

```diff
@@ -605,6 +606,41 @@ func (vcd *TestVCD) removeLeftoverEntities(entity CleanupEntity) {
 
 		vcd.infoCleanup(removedMsg, entity.EntityType, entity.Name, entity.CreatedBy)
 		return
+	case "lbServiceMonitor":
+		if entity.Parent == "" {
+			vcd.infoCleanup("removeLeftoverEntries: [ERROR] No parent specified '%s'\n", entity.Name)
+			return
+		}
+
+		splitParent := strings.Split(entity.Parent, "|")
```

Comment:
Perhaps we should improve the function `splitParent`to handle this case?

Reply:
Addressed